apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -384,12 +424,14 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
                               type="number"
                               className="border-default my-0 block w-16 text-sm [appearance:textfield] ltr:mr-2 rtl:ml-2"
                               placeholder="30"
+                              disabled={periodTypeLocked.disabled}
                               {...formMethods.register("periodDays", { valueAsNumber: true })}
                               defaultValue={eventType.periodDays || 30}
                             />
                             <Select
                               options={optionsPeriod}
                               isSearchable={false}
+                              isDisabled={periodTypeLocked.disabled}
```

Comment:
This was removed recently to allow these fields to be unlocked for v1, now with v2 we need them as the team admin can discretely select their state